### PR TITLE
Complete exercise 1

### PR DESCRIPTION
**Pull request by DavideDeFeudis**
made array of students name and age.

### CODE REVIEW

**Comment by DavideDeFeudis** [comment]
Please review

**Review by Givaramahfoud — COMMENTED** [review]
nice work habibi

File: revisions.js [hunk 1]
```diff
@@ -105,25 +144,89 @@ const classMates = [
...
+// console.log(includeStudent);
+
+// false || false is false
+// true || false is true; if there is 1 true is not going to check the next
+
+// const includeStudent = classMates.some(v => v.firstName === 'Mojgan')
+// console.log(includeStudent);
+
+// const includeStudent = classMates.filter(v => v.firstName === 'Mojgan')
+// console.log(includeStudent);
```
Comment (itamargiv) [inline revisions.js]: Perfect solution! What if I wanted to check that all the students are over 18? :thinking: Also, I would try to use more descriptive variables, at least at the beginning, it helps you collaborate with other people on your code.

File: revisions.js [hunk 2]
```diff
@@ -105,25 +144,89 @@ const classMates = [
...
+
+
+
+
+
+
+
+
+
+
```
Comment (itamargiv) [inline revisions.js]: Whoa! What happened here ;)

**Review by itamargiv — APPROVED** [review]
Great job! :100: I added a few exercises and bonus tasks to my branch, you can check them out if you want.

**Comment by DavideDeFeudis** [comment]
Submit code to review.

**Comment by DavideDeFeudis** [comment]
Please review.

**Pull request merged**